src/Serilog.Extensions.Logging/Serilog.Extensions.Logging.csproj

```diff
@@ -28,6 +28,8 @@
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'net461' or '$(TargetFramework)' == 'netstandard2.0' ">
+    <PackageReference Include="Microsoft.AspNetCore.Hosting.Abstractions" Version="2.0.0" />
```

Comment:
This one's a bit unfortunate, as David points out. Still may not be much of a problem in practice. Could split `UseSerilog()` out to a separate package, if we want to...?